eslint.config.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+ module.exports = {
+	env: {
+        browser: true
+	},
+	extends: [require.resolve('eslint-config-liferay'), require.resolve('eslint-config-prettier')],
```

Comment:
I don't think you need the Prettier reference here, because `eslint-config-liferay` already [gives you that](https://github.com/liferay/eslint-config-liferay/blob/0e10ac76542c3f245fb6b22a698168b770b79dda/index.js#L16). Also, you shouldn't need or use the `require.resolve` approach: just `'liferay'` should work, because it is brought in as a dependency because you declared it in the `package.json`.

Reply:
Good catch, I didn't know it!